plugins/flytekit-k8s-pod/README.md

```diff
@@ -1,5 +1,10 @@
 # Flytekit Kubernetes Pod Plugin
 
+> [!IMPORTANT]
+> This plugin is no longer needed and is here only for backwards compatibility. No new versions will be published after v1.13.x
+> Please use the `pod_template` and `pod_template_name` args to `@task` instead.
```

Comment:
Can we include a link to https://docs.flyte.org/en/latest/deployment/configuration/general.html#configuring-task-pods-with-k8s-podtemplates ?